README.md

```diff
@@ -20,6 +20,16 @@ Usage of wasmserve
         Build tags
 ```
 
+# Trigger Refresh
+
+Once the browser loads the page, you can trigger a reload by making a call to teh server at `/__notify`, like this:
```

Comment:
`_notify`?

Reply:
oh, looks like I added this twice... and fixed one... 😂